cetmix_tower_server/models/cx_tower_plan_line.py

```diff
@@ -44,7 +57,12 @@ class CxTowerPlanLine(models.Model):
         help="Actions trigger based on command result."
         " If empty next command will be executed",
     )
-    command_code = fields.Text(related="command_id.code", readonly=True)
+    command_code = fields.Text(
+        related="command_id.code",
+        comodel_name="cx.tower.command",
+        string="Code",
```

Comment:
no need to add "string" parameter here, because it will be automatically generated from the field name.